running-coreos/cloud-providers/vultr/index.md

```diff
@@ -35,8 +55,11 @@ CoreOS is designed to be [updated automatically]({{site.url}}/using-coreos/updat
 
 <pre>#!ipxe
 
+# Location of your shell script.
+set cloud-config-url http://example.com/cloud-config-bootstrap.sh
```

Comment:
I'm thinking a HEREDOC will have us covered since that can define the file name.